bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerManager.java

```diff
@@ -146,21 +144,21 @@
      * @param failureRc
      *          Failure RC code passed to finalCb when exceptions occured.
      */
-    public void asyncProcessLedgers(Processor<Long> processor, AsyncCallback.VoidCallback finalCb,
+    void asyncProcessLedgers(Processor<Long> processor, AsyncCallback.VoidCallback finalCb,
                                     Object context, int successRc, int failureRc);
 
     /**
-     * Loop to scan a range of metadata from metadata storage
+     * Loop to scan a range of metadata from metadata storage.
      *
      * @return will return a iterator of the Ranges
      */
-    public LedgerRangeIterator getLedgerRanges();
+    LedgerRangeIterator getLedgerRanges();
 
-    /*
+    /**
      * Used to represent the Ledgers range returned from the
      * current scan.
      */
-    public static class LedgerRange {
```

Comment:
Do not remove static

Reply:
`static` is implied in an interface. Reverting that change results in:

    [ERROR] /Users/acoburn/Projects/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerManager.java:161:5: Redundant 'static' modifier. [RedundantModifier]